requirements.sh

```diff
@@ -17,19 +17,62 @@
 ##--------------------------------------------------------------------
 
 ##
-## Author: Ashish Jabble
+## Author: Ashish Jabble, Massimiliano Pinto
 ##
 
 
 set -e
 
-sudo apt update
-sudo apt -y upgrade
+foglamp_location=`pwd`
+is_rhel=`(lsb_release -ds 2>/dev/null || cat /etc/*release 2>/dev/null | head -n1 || uname -om) | egrep '(Red Hat|CentOS)' || echo ""`
+if [ "${is_rhel}" != "" ]; then
+	echo "Platform is ${is_rhel}"
+	sudo yum check-update
+	sudo yum update -y
+	sudo yum-config-manager --enable 'Red Hat Enterprise Linux Server 7 RHSCL (RPMs)'
```

Comment:
is_rhel is a generic check for all RHEL and CentOS, however this is not available on rhel 8 and centos and the script will fail in case this runs on these environments
``` Error: No matching repo to modify: Red Hat Enterprise Linux Server 7 RHSCL (RPMs).```